hook-janitor/src/webhooks.rs

```diff
@@ -177,12 +182,39 @@ impl WebhookCleaner {
         })
     }
 
+    async fn reschedule_stuck_jobs(&self) -> Result<u64> {
+        let mut conn = self
+            .pg_pool
+            .acquire()
+            .await
+            .map_err(|e| WebhookCleanerError::AcquireConnError { error: e })?;
+
```

Comment:
Worth adding a Harry-comment about:
- why we don't increment `attempt` when unlocking rows. My first instinct was to do so, but it's not desired
- what happens if for some reason the previous pod comes back to life and finishes it: I'm assuming that we'll duplicate the output, and record the success one or twice (depending on whether janitor runs between both updates?). Again, it's fine if that's the case for now (the events already have dupes from processing), just would love a quick list of tradeoffs while it's fresh in your head.